docs/Getting Started/Installation/index.md

```diff
@@ -189,3 +189,84 @@ from within the JavaEE directory and a JAR should be generated in the build/libs
+
+## Node.js Setup
+This build allows you to create apps that run on your computer. [Node.js](https://nodejs.org/en/) and npm are the tools you will need to integrate this library into what will be your app.
+In a new directory, save your `SDL.min.js` build there, and then create a new file in the same directory named `index.js`. Make the contents of that file the following:
+```js
+const SDL = require('./SDL.min.js');
+console.log(SDL);
+```
+Finally, run the file: `node index.js`. You should see the SDL library version be printed and the imported SDL object. If so, then you have successfully set up the SDL library!
+
+## WebEngine App Setup
+WebEngine apps use the vanilla JavaScript build, and are set up in a similar fashion to JS apps where it will also run in the browser. Set up the `index.js` and `index.html` file like in the **Vanilla JavaScript Setup** section. The majority of the configuration for the app will now be separated into a `manifest.js` file and then imported into the `index.html` file. Create a `manifest.js` file like below and save it in the same directory as the other two files. The entrypoint's value should have the same name as your app's HTML file. 
+```js
+export default {
+    "entrypoint": "./index.html",
+    "appIcon": "./app_icon.png",
+    "appId": "hello-webengine",
+    "appName": "Hello WebEngine",
+    "category": "DEFAULT",
+    "additionalCategories": [],
+    "locales": {
+        "de_DE": {
+            "appName": "Hallo JS",
+            "appIcon": "./app_icon.png"
+        }
+    },
+    "appVersion": "1.0.0",
+    "minRpcVersion": "6.0",
+    "minProtocolVersion": "5.0"
+};
+```
+Note that the manifest file is using the import/export module syntax; in the HTML file it should be imported in as a module:
+```js
+import sdl_manifest from './manifest.js';
+```
+Also note that in the transport configuration the parameters for `WebSocketClientConfig` will be empty. This is because for WebEngine apps those connection details are expected as query parameters in the URL. See below for an example of what the URL is expected to be once the server is running. `sdl-host` is the location of the SDL Core WebSocket server. `sdl-port` is the port of that WebSocket server. `sdl-transport-role` refers to SDL Core's role, which is as a server (as opposed to a client).
+```
+http://localhost:3000/?sdl-host=HOST&sdl-transport-role=ws-server&sdl-port=PORT
+```
+
+### Building The JavaScript Library    
```

Comment:
I think this may answer my "where is the sdl.min.js" questions. Should it be up top? If devs are just supposed to be using releases, I don't think this section is necessary, and can just be in the README.

Reply:
Removing this section and clarifying the name of the build files at the start of the doc.